### PR TITLE
chore: add snippets for queue ids.

### PR DESCRIPTION
**Pull request by raksiv**

### CODE REVIEW

**Review by davemooreuws — APPROVED** [review]

File: docs/reference/api/queues/send-tasks.mdx [hunk 1]
```diff
@@ -39,6 +39,38 @@ Using one of the provided Nitric SDKs for your language is the easiest way to wo
...
+	},
+	{
+		label: "Go",
+		value: "go"
+	}
+	]}
+	defaultLang="node"
+>
+
+<CodeSnippet snippet="nitrictech/node-sdk/examples/queues/send-id.ts" />
```
Comment (davemooreuws) [inline docs/reference/api/queues/send-tasks.mdx:64]: incorrect according to: https://github.com/nitrictech/node-sdk/pull/77/files

**Review by davemooreuws — CHANGES_REQUESTED** [review]
Inconsistent with node-sdk PR

**Comment by davemooreuws** [comment]
related to: https://github.com/nitrictech/node-sdk/pull/77

File: docs/reference/api/queues/send-tasks.mdx (hunk 1 above)
Comment (raksiv) [reply docs/reference/api/queues/send-tasks.mdx]: updated filename

**Review by davemooreuws — APPROVED** [review]

**Review by tjholm — APPROVED** [review]

**Pull request merged**